### ISSUE TITLE
Header not up to Material Design spec?

### ISSUE THREAD

**Issue by yarian** [GeekyAnts/NativeBase#1550]
## react-native, react and native-base version
react-native `0.52.0`
react `16.2.0`
native-base `^2.3.3`

## Reproduction

```jsx
      <Header>
        <Left>
          <Button onPress={this.toggleHamburger} transparent>
            <Icon name="menu" />
          </Button>
        </Left>
        <Body>
          <Title>Settings</Title>
        </Body>
        <Right></Right>
      </Header>
```

I am getting results that don't line up with [Material UI Guidelines on headers](https://material.io/guidelines/layout/structure.html#structure-app-bar).

## Actual Behavior

<img width="700" alt="screen shot 2018-01-29 at 10 55 18 am" src="https://user-images.githubusercontent.com/596223/35528655-673ef146-04e3-11e8-9056-dc01158bb6ce.png">

## Expected behaviour
I expect Left to not be so large and Title to line up with the 72dp guideline.

## Is the bug present in both ios and android or in any one of them?
Android

**Comment by yarian** [comment]
Looked like a relevant issue–not sure why it was closed: https://github.com/GeekyAnts/NativeBase/issues/795

**Comment by SupriyaKalghatgi** [comment]
@yarian What is the tool you used to measure this?

**Comment by yarian** [comment]
I copied a screenshot unto Sketch.

I ended up getting the correct guidelines by adding the following styles to different components in the Header:

```
{flex: 0, paddingLeft: 6, width: 62}
```
To the `Left` component.

```
{flex: 1}
```
To `Body`

```
{flex: 0}
```
To `Right`

I add `paddingLeft: 6` because Header by default comes with 10dp of padding on both sides, which seems a bit weird.

**Comment by rohithramachandran** [comment]
Same happening for me too. @yarian s solution is not working for me . Any other fixes for this @SupriyaKalghatgi ?

**Comment by sanyesz123** [comment]
@yarian Add **block** attribute too.

**Comment by wedneyyuri** [comment]
@yarian I also can't understand why the issue https://github.com/GeekyAnts/NativeBase/issues/795 was closed. The expected behavior is to follow material design specs without any additional customization.

**Comment by wedneyyuri** [comment]
> Same happening for me too. @yarian s solution is not working for me . Any other fixes for this @SupriyaKalghatgi ?

@rohithramachandran  Are you sure? It's working for me (using v2.4.1). 

```jsx
<Root>
    <Container>
        <Header style={styles.topMargin}>
            <Left style={{ backgroundColor: "green", flex: 0, flexBasis: 48 }}>
                <Button transparent>
                    <Icon name="more" />
                </Button>
            </Left>
            <Body style={{ backgroundColor: "#00adef", flex: 1 }}>
                <Title>Awesome App With Very Very Long Title</Title>
            </Body>
            <Right style={{ backgroundColor: "green", flex: 0, flexBasis: 48 }}>
                <Button transparent>
                    <Icon name="more" />
                </Button>
            </Right>
        </Header>
    </Container>
</Root>
```
![image](https://user-images.githubusercontent.com/7511692/38457234-00b7f830-3a64-11e8-89ee-6a463b140d92.png)
![image](https://user-images.githubusercontent.com/7511692/38457242-16d72afa-3a64-11e8-9691-94ea8fbf70c5.png)

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.4.3](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.4.3)
